setup.cfg

```diff
@@ -16,5 +16,8 @@ classifier =
         Programming Language :: Python
 
 [files]
-packages =
-    stackhpc_monasca_agent_plugins
```

Comment:
If we remove this, does it prevent us from importing from stackhpc_monasca_agent_plugins in a check or detect script?

Reply:
I think so. I removed it because we don't need it for the current plugins and having duplicate files installed might be confusing. I did think about a symlink from the package directory to the monasca plugins directories, but pbr was preventing my custom install step from running. I'm sure there's a way to get that to work, but I figured that we'll likely want to upstream these, so it wasn't worth spending too much time on. I can re-instate it if you think it's worth it?

Reply:
No, let's go with this.